Core/src/main/java/org/tribuo/evaluation/EvaluationAggregator.java

```diff
@@ -174,6 +174,27 @@ public static <T extends Output<T>, R extends Evaluation<T>> Map<MetricID<T>, De
         return results;
     }
 
+    /**
+     * Summarize all fields of a list of evaluations produced by {@link CrossValidation}.
+     * @param evaluations The evaluations to summarize.
+     * @param <T> The output type.
+     * @param <R> The evaluation type.
+     * @return The descriptive statistics for each metric.
+     */
+    public static <T extends Output<T>, R extends Evaluation<T>> Map<MetricID<T>, DescriptiveStats> summarizeCrossValidation(List<Pair<R, Model<T>>> evaluations) {
```

Comment:
This change is separate from the rest of them but I added it to make it easier to aggregate across cross validation folds which is currently a little ugly to do and it didn't feel worth a full PR.